mantidproject/mantid
# IndirectCorrections Outputworkspace name only contains _shifted when shift used

Fixes #14518

_shifted should only be appended to the processing workspaces if a shift is actually being used.

_Data can be obtained from `\\olympic\Babylon5\Public\ElliotOram\Corrections\ContainerSubtraction\IRIS\Base`_
# To Test
- Open ContainerSubtraction ( Interfaces > Indirect > Corrections > ContainerSubtraction)
- Sample=`irs26176_graphite002_red.nxs`
- Container=`irs26174_graphite002_red.nxs`
- Click `Run` 
- Ensure non of the workspaces in the ADS contain `_shifted` in their name.
- Check `Shift x-values of container by adding` and set the value to non be 0
- Click `Run`
- Ensure the workspaces in the ADS that contain `26174` have `_shifted` in the name (excluding the workspaces containing the word `Subtract`)
